website/integrations/services/apple/index.md

```diff
@@ -0,0 +1,327 @@
+>
+> Organizations using Apple Business Essentials can allow their users to authenticate into their Apple devices >using their IdP credentials, typically their company email addresses.
+>
+> -- [Apple Business Manager](https://www.apple.com/business/)
+
+:::info Apple Device Management Platforms
+
+Apple repackages their device management platform into three brands to cater to different audiences:
+
+- Apple Business Manager: Large organizations
+- Apple Business Essentials: Small businesses
+- Apple School Manager: Educational institutions
+
+While this integration guide focuses on Business Manager, the instructions are applicable to all three platforms with minor changes to the terminology.
+
+:::
+
+## Authentication Flow
+
+This sequence diagram shows a high-level flow between the user's MacBook, authentik, and Apple.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant authentik
+    participant Apple
+
+    User->>Apple: User sign-in via iCloud
+    Note over Apple: 🔍 Email domain is federated
+    Apple-->>authentik: Redirect to authentik
+    Note over authentik: ✅ Authenticate
+    authentik-->>Apple: Identity verified
+    Apple-->>User: Device enrolled!
+
+```
+
+In short, Apple Business Manager recognizes the email domain
+as a federated identity provider controlled by authentik. When a user signs in with their email address, Apple redirects them to authentik for authentication. Once authenticated, Apple enrolls the user's device and grants access to Apple services.
+
+---
+
+## Preparation
+
+By the end of this integration, your users will be able to enroll their Apple devices using their authentik credentials.
+
+You'll need to have authentik instance running and accessible on an HTTPS domain, and an Apple Business Manager user with the role of Administrator or People Manager.
+
+:::warning Caveats
+
+Be aware that Apple Business Manager imposes the following restrictions on federated authentication:
+
+- Federated authentication should use the user’s email address as their username. Aliases aren’t supported.
+- Existing users with an email address in the federated domain will automatically be converted to federated authentication, effectively _taking ownership_ of the account.
+- User accounts with the role of Administrator, Site Manager, or People Manager can’t sign in using federated authentication; they can only manage the federation process.
+
+:::
+
+### Placeholders
+
+The following placeholders are used:
+
+- `authentik.company`: The FQDN of the authentik installation.
+
+## authentik configuration
+
+Before proceeding with the Apple Business Manager configuration, you will to need to configure authentik to act as an identity provider for Apple. Start by logging into your authentik instance and navigating to the Admin interface.
+
+### 1. Scope mapping
```

Comment:
When I mentioned numbered steps, I meant the actual steps below each main section. So starting with line 84., it wold be "1. In the Admin interface, navigate to...."